tests/test_conda_subprocess.py

```
@@ -42,6 +43,20 @@ def test_check_output_name(self):
             expected_output,
         )
 
+    def test_check_output_env_name(self):
+        expected_output = "py313"
+        self.assertEqual(
+            check_output("which python", prefix_name=self.env_name, universal_newlines=True),
```

Ruff (0.8.2)

Line too long (93 > 88)

(E501)